libraries/ms-helper/inc/thermistor.h

```diff
@@ -1,12 +1,7 @@
 #pragma once
-// Thermistor Reading interface for NXRT15XH103FA1B___
-// src:
-// https://www.murata.com/en-global/products/productdetail.aspx?cate=luNTCforTempeSenso&partno=NXRT15XH103FA1B___
+// Thermistor Reading interface for NXRT15XH103FA1B (10k Ohm NTC)
 // Requires ADC, GPIO and Interrupts to be initialized.
 #include "adc.h"
-#include "gpio.h"
-#include "interrupt.h"
-#include "soft_timer.h"
 
```

Comment:
Why did you remove the `gpio` header?

Reply:
`adc` already has `gpio` in its header so including it here would be redundant no?

Reply:
It makes it explicit that GPIO is used in this module specifically since you should be setting up the pin. It also means that this module doesn't depend on `adc.h` actually including `gpio.h`. If we ever removed `gpio.h` from `adc.h`, this module would suddenly fail.